code/modules/recycling/disposal.dm

```diff
@@ -1040,12 +1041,15 @@
 /obj/structure/disposalpipe/sortjunction/Initialize(mapload)
 	. = ..()
 	updatedir()
-	if(sort_type_txt)
-		var/list/sort_type_str = splittext(sort_type_txt, ";")
+	var/list/sort_type_str = splittext(sort_type_txt, ";")
+	if(length(sort_type_str) && !("1" in sort_type_str)) // Default to disposals if mapped with it along other destinations
```

Comment:
Might want to make a `stack_trace` when it is mapped incorrectly. As this should also fail the CI builds we got.

Reply:
I added the stack trace and split it into a separate proc